website/docs/wallet/withdraw-validator.md

```diff
@@ -16,87 +18,68 @@ import TabItem from '@theme/TabItem';
-
-Validator exits are irreversible, and so are full validator withdrawals. Once you exit, you cannot do anything with your validator except for withdraw it. Once you perform a full withdrawal, you will receive your validator balance at the Ethereum address of your choosing, and you cannot revert this action. You could use your withdrawn funds to spin up another separate validator if you wish, however.
-
-### Partial withdrawals do not exit your validator
-
-Partial withdrawals only withdraw your validator earnings to an Ethereum address of your choosing, but do not exit nor require an exit from your validator. That is, if you are running a validator that has a balance of 33.3 ETH, you can perform a bls-to-execution-change request to initiate the partial withdrawal of 1.3 ETH and automatic withdrawals of earnings over 32ETH in the future. Your validator will continue to operate normally
-
-### Partial withdrawals of your earnings will continue indefinitely to your Ethereum address of your choice
+- **Pool:** Upon submission of a validator exit request or bls-to-execution-change request, the message will sit in a special place in memory ( the pool ) to be broadcasted across your peers. Since only the block proposers can include these requests and there is a limit to the number of requests included per block, sometimes if the pool becomes too full your message may be dropped and not included. If this happens, a re-submission of the request may be required.
 
-Once a partial withdrawal is included on-chain, earnings will continue to accrue in the Ethereum address you initially specified. This address **cannot be changed** once set, so you must ensure it is protected and one you have custody over. **note: there is only one withdrawal address and this is the same address that will be used for full withdrawals
+## Important guidelines
 
-### Smart contract addresses can be used as the withdrawal address but can not trigger functions.
+When withdrawing your validator, keep the following guidelines in mind:
 
-Smart contract addresses set as the withdrawal credential will only update balances and not trigger any logic. Read [https://eips.ethereum.org/EIPS/eip-4895](https://eips.ethereum.org/EIPS/eip-4895) for more reasoning.
+ 1. **Validators must be fully exited to trigger a full withdrawal:** Full validator withdrawals require your validator to exit first, as exits do not happen automatically. You will need to submit a voluntary exit by following our documentation [here](exiting-a-validator.md). Once your validator exits, it will no longer need to perform its responsibilities after some time (there can be a delay if the validator is part of a sync committee or recently slashed) . The ordering of requests for setting withdrawal credentials or exiting does not matter, once a validator has both its withdrawal credentials updated as well as in an exited state funds will automatically be added to the chosen execution address when processed.
```

Comment:
Intent here is to minimize header bloat and then, in a follow-up PR, we can tighten these guidelines up to make the list even more consumable. The fewer section headers we have, the less overwhelming the content will be, which could help with retention/focus/comprehension. Succinct, minimal copy is very important for a document like this - it'll never be perfect, but we can iterate continuously, one improvement at a time.